tests/test-skeleton/sympy-0.7.5/meta.yaml

```diff
@@ -4,7 +4,7 @@ package:
 
 source:
   fn: sympy-0.7.5.tar.gz
-  url: https://pypi.python.org/packages/source/s/sympy/sympy-0.7.5.tar.gz
+  url: https://pypi.python.org/packages/8c/a5/5fa8adee81837687f7315122769fc0b0e8b042c69e2fe5809c41191c7183/sympy-0.7.5.tar.gz
```

Comment:
I don't know when this changed, but pypi changed their web api.  They don't seem to offer redirects that I know of.  My search terms have been "pypi url change" and stuff like that.  Haven't been able to find anything.